src/main/kotlin/com/redhat/devtools/intellij/kubernetes/tree/ResourceWatchController.kt

```diff
@@ -26,18 +28,27 @@ object ResourceWatchController {
 		tree.addTreeExpansionListener(object: TreeExpansionListener {
 			override fun treeExpanded(event: TreeExpansionEvent) {
 				val descriptor = getDescriptor(event)
+				val telementry = createTelemetry(descriptor)
```

Comment:
Rename as telemetry

Reply:
corrected the spelling error "telementry" -> "telemetry", thanks for spotting.